src/signals/incident-management/containers/FilterTagList/index.js

```diff
@@ -15,33 +15,36 @@ const FilterWrapper = styled.div`
 const StyledTag = styled(Tag)`
   display: inline-block;
   margin: 0 5px 5px 0;
-`;
+  :first-letter {
+    text-transform: capitalize;
+  }`;
 
 export const allLabelAppend = ': Alles';
 
-const renderTag = (key, mainCategories, list) => {
-  let found = false;
-
-  if (list) {
-    found = list.find(i => i.key === key || i.slug === key);
-  }
-
-  let display = (found && found.value) || key;
-
-  if (!display) {
-    return;
-  }
+export const FilterTagListComponent = props => {
+  const {
+    tags,
+    dataLists,
+    categories: { main, sub },
+  } = props;
 
-  if (moment(display, 'YYYY-MM-DD', true).isValid()) {
-    display = moment(display).format('DD-MM-YYYY');
-  }
+  const map = {
+    ...dataLists,
+    maincategory_slug: main,
+    category_slug: sub,
+  };
 
-  const foundMain = mainCategories.find(i => i.key === key);
+  const mapKeys = useCallback(key => {
```

Comment:
This function doesn't seem to have a test. Can you add one? Also, the `useCallback` function doesn't have any dependencies. Would be better to take this function out of the context of `FilterTagListComponent`